etc/module.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="Inchoo_ProductFAQ" setup_version="1.0.0">
+        <sequence>
+            <module name="Magento_Catalog"/>
+            <module name="Magento_Customer"/>
+            <module name="Magento_Store"/>
```

Comment:
koristio si i Magento\Ui, Magento\Cms (Framework i Backend nemoraš tu stavljat), kad si gotov sa modulom, odeš na search unutar foldera gdje je modul i upišeš "Magento\" da vidiš jel si koristio što od magento modula koje nisi stavio u modul xml. Ako si koristio neki drugi namespace od neke ekstenzije osim magenta, napraviš to isto. promjenit će ti se load order na temelju ovoga

Reply:
dodani Magento\Ui i Magento\Cms. koliko vidim jedini su nedostajali. u startu sam stavio ova tri, pa sam zaboravio nadopuniti